src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -768,11 +768,13 @@ private void DynamoViewModelChangeScaleFactor(object sender, EventArgs e)
             var view = new Prompts.ChangeScaleFactorPrompt(dynamoViewModel.ScaleFactorLog) { Owner = this };
             if (view.ShowDialog() == true)
             {
-                if (dynamoViewModel.ScaleFactorLog != view.SliderValue)
+                if (dynamoViewModel.ScaleFactorLog != view.ScaleValue)
                 {
-                    dynamoViewModel.ScaleFactorLog = view.SliderValue;
+                    dynamoViewModel.ScaleFactorLog = view.ScaleValue;
                     dynamoViewModel.CurrentSpace.HasUnsavedChanges = true;
 
+                    Log("Scale factor log changed to " + view.ScaleValue);
```

Comment:
@aparajit-pratap - I agree with @gregmarr. The commas after the decimal point are strange and there should be an additional comma in "1000,000" for en-us. Is it possible to fix this?